### DISCUSSION TITLE
Datatable: customize select all checkbox in header

### DISCUSSION THREAD

**Discussion by YotillaPablo** [primefaces/community#664] [PrimeFaces] [answered]
Is there any way of customizing the select all checkbox in a datatable header?
I want to make it show three states instead of two, but it seems there's no facet for this.

**Comment by melloware** [comment, marked as answer]
I think you have assessed it correctly that you can't change its behavior using a facet.

**Reply by YotillaPablo** [reply]
Ok, I'll see what I can do then. Thanks for answering!
